client/src/components/Page.js

```diff
@@ -12,7 +12,16 @@ import utils from 'utils'
 
 import { showLoading, hideLoading } from 'react-redux-loading-bar'
 import { animateScroll, Link } from 'react-scroll'
-import { setPageProps, setSearchProps, setSearchQuery, clearSearchQuery, DEFAULT_PAGE_PROPS, DEFAULT_SEARCH_PROPS} from 'actions'
+import {
+	setPageProps,
+	setSearchProps,
+	setSearchQuery,
+	clearSearchQuery,
+	DEFAULT_PAGE_PROPS,
+	DEFAULT_SEARCH_PROPS,
+	goToPaginated,
+	resetPages,
```

Comment:
Move `goToPaginated` and `resetPages` to between `clearSearchQuery` and `DEFAULT_PAGE_PROPS`.